src/legacy/core_plugins/kibana/public/index.scss

```diff
@@ -30,3 +30,10 @@
 // Dashboard styles
 // MUST STAY AT THE BOTTOM BECAUSE OF DARK THEME IMPORTS
 @import './dashboard/index';
+
+// helper class that hides elements when rendered in percy
+@media only percy {
+  .hide-in-percy {
```

Comment:
Can you just change this to `.hideInPercy` so it matches our css rules. I swear we're turning the linter on for this stuff over time. @andreadelrio is working on a chunk right now, and it will catch this stuff without us having to remind people of dumb naming things that don't matter.